.circleci/config.yml

```diff
@@ -209,6 +209,9 @@ jobs:
         command: |
           curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./tmp/cc-test-reporter
           chmod +x ./tmp/cc-test-reporter
+    - run:
+        name: Precompile assets
```

Comment:
Could this be a shared job, rather than be repeated?

Reply:
What do we gain from making it a shared job?

Personally I find it harder to jump around the config file (it's over 500 lines now 😬).

How do you feel about a 1 line command? I think it's pretty self-explanatory so we could lose the `name` argument 🤷‍♂️

```yml
- run: bundle exec rails assets:precompile
```

Reply:
If we remove duplication we may be able to get it to 499 lines! 😆 
Fair point, I use an IDE that lets me click through re-used yaml tags *most* of the time so I get less affected by the file size